analysis/src/Xform.ml

```diff
@@ -110,9 +110,20 @@ module IfThenElse = struct
     | Some newExpr ->
       let range = rangeOfLoc newExpr.pexp_loc in
       let newText = printExpr ~range newExpr in
+      let uri = Uri.fromPath path |> Uri.toString in
```

Comment:
To return `uri` scheme.

Reply:
Forgot why this does not just the string it starts with?

Reply:
Was breaking code actions in neovim client because it was not returning uri scheme from LSP spec.

Example: Add type annotation
![image](https://user-images.githubusercontent.com/16160544/182999584-46b4dc4e-247a-4f3f-adf8-759f562b58c7.png)

```json
[{"title": "Add type annotation", "kind": "refactor.rewrite", "edit": {"documentChanges": [{
  "textDocument": {
  "version": null,
  "uri": "/home/pedro/Desktop/learning-rescript/src/intro/Codeaction.res"
  },
  "edits": [{
  "range": {"start": {"line": 20, "character": 5}, "end": {"line": 20, "character": 5}},
  "newText": ": int"
  }]
  }]}}]
```

In vscode it works without problem

Reply:
Sorry what I meant to ask is: what does this composition of functions do?
Let's put it into a single function with a name suggesting what it does, and use that function instead. So it's less likely that the same issue will pop up somewhere else by copy paste and change in future.